rfcs/image-optimization.md

```diff
@@ -0,0 +1,276 @@
+---
+date: 2025-12-16
+---
+
+# Image Optimization
+
+## TL;DR
+
+* **We already partially use Vercel image optimization** via `next/image` in layout components (hero, header, featured images, logos).
```

Comment:
i guess what i don't get is do we get *any* optimization from vercel image if we do *not* know image actual sizes (which i thought we don't - and i thought we'd need cloudflare images or whatever to have b/c it would do the resizing).

If we do get a benefit then: yes, let's do just option 1 for now as super fast and easy. then we can look at part 2 later.

Reply:
this is a great clarification and should go into the rfc. we are solving 2 issues here:

- preventing layout shifts
- reducing size of images served to users (good for their experiences and web vitals).

Let's distinguish this a bit in the rfc.